mkt/reviewers/views.py

```diff
@@ -419,25 +419,21 @@ def _review(request, addon, version):
     return render(request, 'reviewers/review.html', ctx)
 
 
-@transaction.commit_manually
```

Comment:
While you are at it, can you remove the following in `mkt/reviewers/tests/test_views.py`:
```
if os.environ.get('JENKINS_HOME'):
    raise SkipTest()
```